hls-test-utils/src/Test/Hls.hs

```diff
@@ -339,7 +348,8 @@ mkPluginTestDescriptor' pluginDesc plId _recorder = IdePlugins [pluginDesc plId]
 -- @
 pluginTestRecorder :: Pretty a => IO (Recorder (WithPriority a))
 pluginTestRecorder = do
-  initialiseTestRecorder ["HLS_TEST_PLUGIN_LOG_STDERR", "HLS_TEST_LOG_STDERR"]
+  initialiseTestRecorder ["HLS_TEST_PLUGIN_LOG_STDERR",
+        "LSP_TEST_LOG_STDERR", "HLS_TEST_HARNESS_STDERR", "HLS_TEST_LOG_STDERR"]
```

Comment:
I don't think all these environment variables should be used to enable logging for plugins. I don't think this should be changed.